lib/models/lcr-carrier-set-entry.js

```diff
@@ -0,0 +1,65 @@
+const Model = require('./model');
+const {getMysqlConnection} = require('../db');
```

Comment:
We have older code that does it this way 
```
const {getMysqlConnection} = require('../db');
return new Promise((resolve, reject) => {
      getMysqlConnection((err, conn) => {
```
but newer code instead (simpler) does this pattern:
```
const {promisePool} = require('../db');
..
const [r] = await promisePool.query('SELECT *...'
```
For all new code like this, can you follow the latter pattern?  
(When we have time we should do a PR to change all older code as well to this pattern)

Reply:
fixed